jaxrs-spec/src/main/asciidoc/chapters/appendix/_changes-since-3.1-release.adoc

```diff
@@ -14,4 +14,9 @@
 * <<javaee>>: Removal of `@ManagedBean`
 annotation support.  Managed bean support is now part of CDI.
 * <<java-se>>: Clarified when the two alternative ways of publishing on 
-Java SE must be supported.
\ No newline at end of file
+Java SE must be supported.
+* Removed `Link.JaxbLink` and `Link.JaxbAdapter` inner classes.
+* Added `getHeaderString()` method to `ClientRequestContext`, `ClientResponseContext`,
```

Comment:
The `getHeaderString()` should be `containsHeaderString()`.

Reply:
ARGH!  yes, thanks